client/verta/verta/registry/_verta_model_base.py

```diff
@@ -55,7 +56,7 @@ def __init__(self, artifacts):
         raise NotImplementedError
 
     @abc.abstractmethod
-    def predict(self, input):
+    def predict(self, input, headers=Dict[str, str]):
```

Comment:
That's a type annotation! It should be after a `:`, not a `=`

```suggestion
    def predict(self, input, headers: Dict[str, str]):
```

Reply:
oops it used to be None that's why. Why does this work???